django/template/engine.py

```diff
@@ -256,3 +260,53 @@ def make_origin(self, display_name, loader, name, dirs):
             return LoaderOrigin(display_name, loader, name, dirs)
         else:
             return None
+
+    def get_template_debug(self, exception, template, token):
```

Comment:
It's `grep -C10 <line_where_error_occurred> <template_source>`.

It could use a docstring.